### PR TITLE
feature(suite): add the pause button to coinjoin status menu

### LINKED ISSUE #6630: Add resume button to CoinJoin `...` settings
User can Pause a CoinJoin by click on progress circle or in `...` 
Snímek obrazovky 2022-10-21 v 9 58 53

However there is missing `Resume` button when is CoinJoin already paused:
Snímek obrazovky 2022-10-21 v 9 58 58

### PR DESCRIPTION
**Pull request by dahaca**
## Related Issue

Resolve [#6630](https://github.com/trezor/trezor-suite/issues/6630)

## Screenshots:

image

### CODE REVIEW

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx [hunk 1]
```diff
@@ -158,6 +159,29 @@ export const CoinjoinStatus = ({
...
+                                <Icon icon="PLAY" size={10} />
+                                <Translation id="TR_RESUME" />
+                            </MenuLabel>
+                        ),
+                        callback: togglePause,
+                        'data-test': `@coinjoin/resume`,
+                        isHidden: !isPaused || isLoading,
+                    },
+                    {
+                        key: 'resuming',
```
Comment (komret) [inline packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx:175]: Clicking the "resuming" item closes the dropdown. I would expect a default cursor, grey text and no action on click. But no big deal :)
chrome-capture-2022-9-31

**Review by komret — APPROVED** [review]

File: packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx (hunk 1 above)
Comment (dahaca) [reply packages/suite/src/components/wallet/CoinjoinSummary/CoinjoinStatus.tsx:175]: 3edbcd55e

**Comment by komret** [comment]
nice!

**Comment by bosomt** [comment]
**QA OK**


**Info:**
 - Suite version: desktop 22.11.0 (91cc9ae7ad79fee426a16d2f2553a2017d390572)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuite/22.11.0 Chrome/100.0.4896.160 Electron/18.3.5 Safari/537.36
 - OS: MacIntel
 - Screen: 1440x900
 - Device: model T 2.5.3 Universal (revision fac4e2f157ca7d0c782b8a55175478e476ea7e28)
 - Transport: bridge 2.0.32

**Pull request merged**